src/software/backend/replay_logging/replay_test.cpp

```diff
@@ -39,17 +39,20 @@ TEST(ReplayTest, test_read_and_write_replay)
     // do a quick sanity check here to assert that the timestamps are monotonically
     // increasing
     double max_timestamp = 0.f;
+    size_t idx           = 0;
```

Comment:
What's the purpose of `idx` exactly? It isn't used to verify, so is it strictly for debugging?

Reply:
yeah it's for debugging, perhaps this isn't necessary but this was a nice-to-have for when I was debugging using python protobuf

Reply:
please add a comment about it then